### ISSUE TITLE
[a11y] QSelect - Autocomplete attribute will render a second input without tabindex=-1

### ISSUE THREAD

**Issue by Evertvdw** [quasarframework/quasar#14378]
### What happened?

When using `autocomplete` attribute on a QSelect a second `input` element is rendered but it does not have `tabindex=-1` attached to it so it will also be focusable when using keyboard navigation.

### What did you expect to happen?

One tab to enter the select and one to leave it again.

### Reproduction URL

https://codepen.io/Mett-codepens/pen/NWMRYpm

### How to reproduce?

Tab through the page, and notice that you have to tab twice within the QSelect to leave it again.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)
